app/_posts/blog/2016-05-29-2016-retrospective-planning-for-2017.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "DEVit 2016 Retrospective, planning for the future"
+description: In this article we go through the DEVit 2016 big moments, fails and all the excitement we experienced, and then we get a sneak peak into what 2017 has in hold for us.
+metaTitle: DEVit 2016 Retrospective, planning for the future
+metaDescription: In this article we go through the DEVit 2016 big moments, fails and all the excitement we experienced, and then we get a sneak peak into what 2017 has in hold for us.
+category : blog
+tags : [retrospective, devit2016, devit2017]
+author: Thanasis Polychronakis
+---
+
+
+## DEVit 2016 is over!
+
+Oh what a blast DEVit 2016 was! No matter how exhausted we were we could not stop the rush of endorphins that flooded our bodies for days after. You see, we, and by 'we' I'm referring to everyone that was at DEVit, managed to make a difference, we participated in making history for our community, for our country. We have successfully positioned Greece in the global landscape of international conferences, helped make DEVit a sustainable yearly conference and had such an amazing time!
```

Comment:
"for our country" -> I know what you want to say but it is to national. Non-greeks don't care about this and will probably be driven away.
"helped make" -> made